modApiServer/src/org/aion/api/server/rpc3/AionChainHolder.java

```diff
@@ -261,4 +269,8 @@ private Repository getRepoByBlockNumber(long blockNumber){
         return this.chain.getRepository()
             .getSnapshotTo(getBlockByNumber(blockNumber).getStateRoot());
     }
+
+    private static boolean isFutureTimestamp(long timestamp) {
+        return timestamp > (System.currentTimeMillis() / 1000 + 1);
```

Comment:
Could we use the `FutureBlockRule` here? It would make the code more maintainable and devs less likely to update one implementation and not the other.

Reply:
updated to use the `FutureBlockRule`